x-pack/plugins/xpack_main/server/routes/api/v1/xpack_usage.js

```diff
@@ -44,11 +40,11 @@ export function xpackUsageRoute(server) {
           ...xpackUsage
         });
       } catch(err) {
-        req.log(['error'], err);
+        req.log(['error'], err); // FIXME doesn't seem to log anything useful if ES times out
```

Comment:
The FIXME comment here seems to be the case for any API that relies on ES, but I want to look into it more later